### PR TITLE
[Doc Issue][Add a note for VSC files for debugging][2867164]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5b4e80fc7c3f2cae1ce9dc6e84e42d0c64dbda81/msteams-platform%2Ftoolkit%2Fprovision.md) | 94 | 64 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4b8ebb69-8b00-4b55-ae25-431662e54441) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1413727494baa05b2d9c64ddfbdc794f04b5cfe2/msteams-platform%2Ftoolkit%2Fprovision.md) | 94 | 64 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/09489f3e-92be-4670-82d1-e756282357c2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75863010a800e3210d79e61b0315d313e3dd9e03/msteams-platform%2Ftoolkit%2Fprovision.md) | 94 | 64 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/98cea11e-1cd6-4949-afe8-ee173d3b5dac) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/provision.md [hunk 1]
```diff
@@ -222,6 +222,9 @@ You can add following configuration snippet to `.fx\configs\config.{env}.json` f
 
 After adding the snippet, add your client secret to related environment variable for Teams Toolkit to resolve the actual client secret during provision.
 
+> [!NOTE]
+> A Teams Toolkit generated project has a pre-defined set of Visual Studio Code debugging tasks in the `.vscode/tasks.json` file. These pre-defined tasks are generated by Teams Toolkit version 4.1.0 or later. Add the configuration snippet in the `.vscode/tasks.json` file.
+
```
Comment (MuyangAmigo) [inline msteams-platform/toolkit/provision.md:227]: I think the key point here is to mention use `.vscode/tasks,json` for customizing debugging tasks (such as use existing bot) and use `.fx\configs\config.{env}.json` is for cloud environments.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a8cb2010cf60aadfbc3431bbfc7f6554be00946f/msteams-platform%2Ftoolkit%2Fprovision.md) | 94 | 64 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f864fad7-c0ab-40a6-a4cc-86e34b4e0d01) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7daeba98902f47cc1bc85a9985375b0f278f7f2b/msteams-platform%2Ftoolkit%2Fprovision.md) | 94 | 64 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0d3d5e6b-ff35-449a-88c1-a522dd548844) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4c4811fe32d559c8d7254139f17abf1ba099d3e9/msteams-platform%2Ftoolkit%2Fprovision.md) | 94 | 64 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d592c9cd-b7f9-42ec-a1b6-b3e1cf0c473d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by MuyangAmigo — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/provision.md](https://github.com/MicrosoftDocs/msteams-docs/blob/93d03c31587edcae6d83d0f207a857e2e7aa5416/msteams-platform%2Ftoolkit%2Fprovision.md) | 94 | 64 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4023d8f6-99d4-4305-8215-d850f4523840) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**